roles/oVirt.image-template/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
The variables in these roles should be properly namespaced.

ovirt_image_path, ovirt_template_cluster, etc.  Having such generic names can potentially pose a problem for other roles.